assignments/prototypes.js

```diff
@@ -14,15 +14,33 @@
   * dimensions
   * destroy() // prototype method -> returns the string: 'Object was removed from the game.'
 */
-
+function GameObject(characterAttributes)
+{
+  this.createdAt = characterAttributes.createdAt;
+  this.dimensions = characterAttributes.dimensions;
+}
+GameObject.prototype.destroy = function() 
+{
+return  `${this.name} was removed from the game.`;
+}
```

Comment:
Your code looks good here, but I would suggest bringing your opening curly brace up to the end of lines 17 and 22. You could also add a semicolon to the end of curly brace on line 25.